### PR TITLE
Added page for Mountain Madness 2021 submissions

### PR DESCRIPTION
**Pull request by SpartanApple**
Added page linked within Mountain Madness 2021 page to submissions.

### CODE REVIEW

File: csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021.html [hunk 1]
```diff
@@ -31,6 +31,10 @@
       <a href="#prize">
         Prizes
```
Comment (modernNeo) [inline csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021.html:32]: this page is not passing the [validator test](https://validator.w3.org/nu/?doc=https%3A%2F%2Fdev.sfucsss.org%2FPR-420%2Fevents%2Fmm%2F2021).

File: csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021.html [hunk 2]
```diff
@@ -31,6 +31,10 @@
...
       </a>
+
+      <a href="/events/mm/2021/submissions">
```
Comment (modernNeo) [inline csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021.html:35]: you forgot `URL_ROOT`

File: csss-site/src/events/mountain_madness/views.py [hunk 3]
```diff
@@ -5,6 +5,11 @@
 TAB = 'events'
 
 
+def mm2021_submissions(request):
+    path = 'mountain_madness/2021/mountain_madness2021_submissions.html'
```
Comment (modernNeo) [inline csss-site/src/events/mountain_madness/views.py:9]: why declare path rather than just pass it to `render`?

**Review by modernNeo — CHANGES_REQUESTED** [review]

File: csss-site/src/events/mountain_madness/views.py (hunk 3 above)
Comment (SpartanApple) [reply csss-site/src/events/mountain_madness/views.py:9]: The validator complained the line was too long so I broke it up.

Comment (modernNeo) [reply csss-site/src/events/mountain_madness/views.py:9]: you can  just make the `render` function call multi-line instead

**Review by modernNeo — CHANGES_REQUESTED** [review]

File: csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021_submissions.html [hunk 4]
```diff
@@ -0,0 +1,215 @@
+{% extends "csss/header.html" %}
+{% load staticfiles %}
+{% block content %}
+{{ block.super }}
+	<link rel="stylesheet" href="{% static 'mountain_madness_static/2021/csss_custom.css' %}">
+	<section id='above-fold'>
+		<header>
+			<h1 class="title is-size-2">Submissions from the Computing Science Student Society Mountain Madness Hackathon 2021</h1>
```
Comment (modernNeo) [inline csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021_submissions.html:8]: I really don't think this needs to be such a mouthful of a title.

also, If you are gonna have `submissions` here, I don't really see a point to also have it at line 22 at `<h2 class="title is-size-2">Submissions</h2>`

File: csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021_submissions.html [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+			<td class="csss_mountain_madness_submission_row">Second Place</td>
+            <td class="csss_mountain_madness_submission_row">Interrupting Chicken</td>
+			<td class="csss_mountain_madness_submission_row">Interrupting Chicken, an annoying typing game that interrupts your typing</td>
+			<td class="csss_mountain_madness_submission_row"><a href="https://github.com/jacksonn-gtc/MountainMadness2021">https://github.com/jacksonn-gtc/MountainMadness2021</a></td>
+		</tr>
+		<tr>
+			<td class="csss_mountain_madness_submission_row">Third Place</td>
+            <td class="csss_mountain_madness_submission_row">The Meme Machine</td>
+			<td class="csss_mountain_madness_submission_row">The Meme Machine, website that shows and allows for creation and sharing of memes</td>
+			<td class="csss_mountain_madness_submission_row"><a href="www.github.com/asim-shrestha/mountain-madness-2021">www.github.com/asim-shrestha/mountain-madness-2021</a></td>
```
Comment (modernNeo) [inline csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021_submissions.html:47]: please fix this link

File: csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021_submissions.html [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+		<div id='building' aria-hidden="true"></div>
+
+		<div id='mountain' aria-hidden="true">
+			<div class='grass'></div>
+			<div class='grass'></div>
+			<div class='grass'></div>
+		</div>
+
+	</section>
+	Please note, these projects made have been made private or changed since submission. Run at your own risk.
```
Comment (modernNeo) [inline csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021_submissions.html:20]: typo?

**Review by modernNeo — CHANGES_REQUESTED** [review]

File: csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021_submissions.html [hunk 7]
```diff
@@ -0,0 +1,215 @@
+{% extends "csss/header.html" %}
```
Comment (modernNeo) [inline csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021_submissions.html:1]: also, please use the CSS of the https://dev.sfucsss.org/PR-420/events/mm/2021 for this page as well.

the website already looks like shit, let's try and not make it worse and maintain some consistency on these special pages 😅

**Review by modernNeo — CHANGES_REQUESTED** [review]

File: csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021_submissions.html (hunk 7 above)
Comment (SpartanApple) [reply csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021_submissions.html:1]: This was consistent with the Fall Hacks Submission page.

File: csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021.html [hunk 8]
```diff
@@ -56,7 +56,7 @@ <h2><a href="https://www.eventbrite.ca/e/mountain-madness-2021-tickets-139106494
       </div>
 
     </section>
-
+<h2>
```
Comment (modernNeo) [inline csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021.html:59]: `h2` it supposed to surround just a string...not an entire `section` tag.

**Review by modernNeo — CHANGES_REQUESTED** [review]

File: csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021.html (hunk 1 above)
Comment (modernNeo) [reply csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021.html:32]: @SpartanApple the validation is still having issues

File: csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021.html (hunk 8 above)
Comment (modernNeo) [reply csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021.html:59]: what is this `h2` tag for?

File: csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021.html [hunk 9]
```diff
@@ -52,8 +56,9 @@ <h2><a href="https://www.eventbrite.ca/e/mountain-madness-2021-tickets-139106494
...
     <section id='info-container'>
+</h2>
```
Comment (modernNeo) [inline csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021.html:61]: what is this `h2` tag for?

File: csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021_submissions.html (hunk 4 above)
Comment (modernNeo) [reply csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021_submissions.html:8]: can you rename this to `CSSS Mountain Madness Hackathon 2021 Submissions`?

File: csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021_submissions.html [hunk 10]
```diff
@@ -0,0 +1,215 @@
...
+		<div id='mountain' aria-hidden="true">
+			<div class='grass'></div>
+			<div class='grass'></div>
+			<div class='grass'></div>
+		</div>
+
+	</section>
+	Please note, these projects may have been made private or changed since submission. Run at your own risk.
+
+	<h2 class="title is-size-2">Submissions</h2>
```
Comment (modernNeo) [inline csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021_submissions.html:22]: may as well delete this as its redundant.

**Review by modernNeo — CHANGES_REQUESTED** [review]

File: csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021.html [hunk 11]
```diff
@@ -50,9 +54,7 @@ <h2><a href="https://www.eventbrite.ca/e/mountain-madness-2021-tickets-139106494
 				<div class='grass'></div>
 				<div class='grass'></div>
       </div>
-
     </section>
-
     <section id='info-container'>
```
Comment (modernNeo) [inline csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021.html:58]: please add `<h2>F.A.Q.</h2>` right above this section tag. I am curious if that tags away the warning.

File: csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021_submissions.html (hunk 10 above)
Comment (modernNeo) [reply csss-site/src/events/mountain_madness/templates/mountain_madness/2021/mountain_madness2021_submissions.html:22]: @SpartanApple forgot this

**Review by modernNeo — CHANGES_REQUESTED** [review]

**Pull request merged**